shimmingtoolbox/cli/check_env.py

```diff
@@ -149,6 +175,23 @@ def get_dcm2niix_version() -> str:
     return version_output
 
 
+def get_sct_version() -> str:
+    """Gets the ``sct`` installation version.
+
+    This function calls ``sct_check_dependencies -short`` and captures the output to
+    obtain the installation version.
+
+    Returns:
+        str: Version of the ``SCT`` installation.
+    """
+    # `sct_check_dependencies -short` returns
+    sct_version: str = subprocess.run(["sct_check_dependencies", "-short"], capture_output=True, encoding="utf-8")
+    assert sct_version.returncode == 0
```

Comment:
Assert should only be used in debugging/tests. Raise an error instead. Asserts can be turned off "at runtime" via configuration. THis isn't fun to debug.